packages/contextual-help/stories/contextual-help.stories.ts

```diff
@@ -0,0 +1,171 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { html, TemplateResult } from '@spectrum-web-components/base';
+
+import '../sp-contextual-help.js';
+import '@spectrum-web-components/link/sp-link.js';
+import { spreadProps } from '../../../test/lit-helpers.js';
+import type { Placement } from '@spectrum-web-components/overlay/src/overlay-types.js';
+
+interface StoryArgs {
+    label: string;
+    variant: 'info' | 'help';
+    headlint: string;
+    placement: Placement;
+    [key: string]: unknown;
+}
+
+export default {
+    title: 'Contextual Help',
+    component: 'sp-contextual-help',
+    args: {
+        label: '',
+        variant: 'info',
+        headline: undefined,
+        placement: undefined,
+    },
+    argTypes: {
```

Comment:
We need to update our generated files, but can you move this to match https://github.com/adobe/spectrum-web-components/blob/main/packages/accordion/stories/args.ts